src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -0,0 +1,361 @@
+        pgm_sym_loads["node"][:n_loads] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][:n_loads] = pp_loads["in_service"]
+        pgm_sym_loads["type"][:n_loads] = LoadGenType.const_power
+        pgm_sym_loads["p_specified"][:n_loads] = const_p_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][:n_loads] = const_p_multiplier * pp_loads["q_mvar"]
+
+        pgm_sym_loads["id"][n_loads : 2 * n_loads] = self._generate_ids("sym_load_const_impedance", pp_loads.index)
+        pgm_sym_loads["node"][n_loads : 2 * n_loads] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][n_loads : 2 * n_loads] = pp_loads["in_service"]
+        pgm_sym_loads["type"][n_loads : 2 * n_loads] = LoadGenType.const_impedance
+        pgm_sym_loads["p_specified"][n_loads : 2 * n_loads] = const_z_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][n_loads : 2 * n_loads] = const_z_multiplier * pp_loads["q_mvar"]
+
+        pgm_sym_loads["id"][-n_loads:] = self._generate_ids("sym_load_const_current", pp_loads.index)
+        pgm_sym_loads["node"][-n_loads:] = self._get_ids("bus", pp_loads["bus"])
+        pgm_sym_loads["status"][-n_loads:] = pp_loads["in_service"]
+        pgm_sym_loads["type"][-n_loads:] = LoadGenType.const_current
+        pgm_sym_loads["p_specified"][-n_loads:] = const_i_multiplier * pp_loads["p_mw"]
+        pgm_sym_loads["q_specified"][-n_loads:] = const_i_multiplier * pp_loads["q_mvar"]
+
+        self.pgm_data["sym_load"] = pgm_sym_loads
+
+    def _create_pgm_input_transformers(self):
+        assert "transformer" not in self.pgm_data
+
+        pp_trafo = self.pp_data["trafo"]
+
+        pgm_transformers = initialize_array(data_type="input", component_type="transformer", shape=len(pp_trafo))
+        pgm_transformers["id"] = self._generate_ids("transformer", pp_trafo.index)
+        pgm_transformers["from_node"] = self._get_ids("bus", pp_trafo["hv_bus"])
+        pgm_transformers["from_status"] = pp_trafo["in_service"]
+        pgm_transformers["to_node"] = self._get_ids("bus", pp_trafo["lv_bus"])
+        pgm_transformers["to_status"] = pp_trafo["in_service"]
+        pgm_transformers["u1"] = pp_trafo["vn_hv_kv"] * 1e3
+        pgm_transformers["u2"] = pp_trafo["vn_lv_kv"] * 1e3
+        pgm_transformers["sn"] = pp_trafo["sn_mva"] * 1e6
+        pgm_transformers["uk"] = pp_trafo["vkr_percent"] * 1e-2
+        pgm_transformers["pk"] = (pp_trafo["vkr_percent"] * 1e-2) * (pp_trafo["sn_mva"] * 1e6)
+        pgm_transformers["i0"] = pp_trafo["i0_percent"] * 1e-2
+        pgm_transformers["p0"] = pp_trafo["pfe_kw"] * 1e3
+        pgm_transformers["winding_from"] = pp_trafo["vector_group"].apply(get_transformer_winding_from)
+        pgm_transformers["winding_to"] = pp_trafo["vector_group"].apply(get_transformer_winding_to)
+        pgm_transformers["clock"] = (pp_trafo["shift_degree"] / 30).astype(np.int8)
```

Comment:
We should also warn user in case of non-clock shift degree values. ie. shift degree values not multiples of 30